src/t8_cmesh/t8_cmesh_examples.c

```diff
@@ -1215,9 +1216,16 @@ t8_cmesh_new_hypercube_pad (const t8_eclass_t eclass, sc_MPI_Comm comm, const do
   t8_cmesh_t cmesh;
   t8_cmesh_init (&cmesh);
 
-  /* We use standard linear geometry */
-  const t8_geometry_c *linear_geom = t8_geometry_linear_new (dim);
-  t8_cmesh_register_geometry (cmesh, linear_geom);
+  if (use_axis_aligned && (eclass == T8_ECLASS_HEX || eclass == T8_ECLASS_QUAD || eclass == T8_ECLASS_LINE)) {
+    /* We use axis aligned geometries */
+    const t8_geometry_c *axis_aligned_geom = t8_geometry_linear_axis_aligned_new (dim);
+    t8_cmesh_register_geometry (cmesh, axis_aligned_geom);
```

Comment:
The axis aligned geometry uses only 2 vertices per tree. This has to be adapted as well

Reply:
Done